device/src/main/resources/application.properties

```diff
@@ -1,6 +1,8 @@
-spring.jpa.hibernate.ddl-auto=update
-spring.datasource.url=jdbc:mysql://localhost:3306/webStore?createDatabaseIfNotExist=true&serverTimezone=UTC
-spring.datasource.username=root
-spring.datasource.password=root
-spring.datasource.driver-class-name=com.mysql.jdbc.Driver
-spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
+spring.data.mongodb.host=localhost
+spring.data.mongodb.port=27017
+spring.data.mongodb.database=DeviceInfoDB
+logging.level.org.springframework.data.mongodb.core.MongoTemplate=DEBUG
+server.port=8081
+config.hostName=http://localhost:8080
```

Comment:
not clear what this hostname mean. Maybe give it a better property name, so it will be clear that it's a hostname of main application.